### ISSUE TITLE
Does not work 

### ISSUE THREAD

**Issue by Antibioticvz** [krunkosaurus/simg#8]
I've got this Error msg
`Uncaught TypeError: Cannot read property 'readJSON' of undefined`

`
import Simg from 'simg';
...
<Button id="convertBtn" 
                        onClick={
                            ()=>{
                                //Here I can see my svg
                                console.log('SIMG %o', document.getElementById('myChart'));

```
                            console.log('SIMG %o', Simg(document.getElementById('myChart')));

                            Simg(document.getElementById('myChart'));

                            Simg.replace();

                            Simg.download('awesome');
                        }
                    }`
```

**Comment by patrickberkeley** [comment]
@Antibioticvz this happens with the npm package, not the bower package.

**Comment by krunkosaurus** [comment]
Hi @Antibioticvz: I just released v1.1.1 which was a fix for Firefox. Just tested npm with a new install and it worked for me. Let me know if you still see an issue.
